isyfact-standards-doc/src/docs/antora/modules/isy-bedienkonzept/pages/bedienkonzept.adoc

```diff
@@ -867,6 +867,138 @@ Beispiel:
+Falls es unklar ist, welche Aktion wichtiger ist, wird die Standardaktion als primäre Aktion angesehen und entsprechend gefärbt.
+* Das <<label>> sollte kurz und selbsterklärend sein.
+* Das Label sollte die auszuführende Aktion widerspiegeln (z.B. Öffnen, Löschen, Speichern, Weiter).
+* Ruft ein Button einen Dialog auf, soll der resultierende Dialog als Titel die Beschriftung des Buttons wiederholen.
+* Die Schaltfläche für Buttons beträgt mindestens 44px x 44px.
+* Die Breite eines Buttons soll sich am enthaltenen Label orientieren.
+* Nebeneinanderstehende Buttons können unterschiedlich breit sein, wenn die jeweiligen Labels unterschiedlich groß sind.
+* Ein Button soll nach Möglichkeit so platziert werden, dass mögliche Beziehungen zu anderen Elementen deutlich werden.
+* Wenn nach einem Klick auf einen Button die Ergebnisaktion nicht sofort ersichtlich ist, soll dies durch ein System-Feedback (z. B. durch eine <<fortschrittsanzeige>>) veranschaulicht werden.
+
+*_Don'ts_*
+
+* Zum Selektieren oder Auswählen von Inhalten werden die Bedienelemente <<radiobutton>> und <<checkbox>> eingesetzt.
+* Zum Ein- oder Ausschalten von Inhalten werden spezielle <<input-switch>> verwendet.
+* Innerhalb von Fließtexten oder zum Verweisen auf externe Inhalte werden <<link,Links>> genutzt.
+* Um ein Frontend farblich nicht zu überladen, sollten die Button-Farben für spezielle Aktionen mit Bedacht eingesetzt werden.
+
+[[inverted-button]]
+==== Invertierte Button
+
+Falls die <<image-buttoncolors,farblich gefüllten Buttons>> zu bunt für das Frontenddesign sind, können auch die invertierten Varianten benutzt werden:
+
+[[image-buttoncolors-inverted]]
+.Farben von invertierten Buttons
+image::24_BE_Outlined_Buttons.png[align="center"]
+
+In der invertierten Button-Variante ist die Button-Farbe für jeden Aktionsbutton weiß und nur die Schrift und die Button-Border farblich eingefärbt.
+Die unterschiedlichen Button-Zustände lassen sich wie bei den farblich gefüllten Buttons voneinander abgrenzen.
+D.h. bei den Zuständen "Hover" und "Pressed" wird die Button-Farbe dunkler und beim Zustand "Disabled" heller.
+Im "aktiven" bzw. "fokussierten" Zustand bekommt der Button einen Fokus-Rahmen, der einen helleren Ton hat als der Button selbst.
+
+[[icon-button]]
+==== Icon-Button
+
+Icon-Buttons sind Buttons die kein sichtbares Label haben, sondern nur aus einem aussagekräftigen <<icons,Icon>> bestehen.
+Sie sind rund und haben einen farbigen Rand, um sich vom Hintergrund oder anderen <<basiselemente,Basis>>- und <<bedienelemente,Bedienelementen>> abzugrenzen.
+Deshalb ist es essenziell, dass das Icon die dahinterstehende Funktionalität ausdrückt.
+Aus diesem Grund sollten Icon-Buttons mit Bedacht eingesetzt werden und können somit nicht jeden beliebigen beschrifteten Button ersetzen.
+Am häufigsten wird diese Button-Art in der Aktionsspalte einer Tabelle benutzt.
```

Comment:
...diese Button-Art in Kombination mit einer Aktion eines Bedienelements oder Designpattern genutzt, wie zum Beispiel der Aktionsspalte einer Tabelle oder zum Hinzufügen oder Löschen von Elementen.